docs/2.7.3.md

```diff
@@ -0,0 +1,31 @@
+# ProxySQL 2.7.3 and ProxySQL admin tools (2025-04-21)
+
+--8<--- "rn-block-top.md"
+
+## Release highlights
+
+In this release, we are skipping [ProxySQL 2.7.2]. We are releasing ProxySQL 2.7.3., which is the latest ProxySQL release. This decision ensures that you receive the most up-to-date features and improvements. Compared to the previous [ProxySQL 2.7.1-1 release], this release includes all improvements and bug fixes available in [ProxySQL 2.7.2] and [ProxySQL 2.7.3].
+
+* Added a new system for logging and analyzing events, providing real-time understanding of how applications are interacting with the database and its performance.
```

Comment:
This would be more appropriate `Added a configurable feature for logging and analyzing events and queries...`